src/templates/gwfcore/gwfcore-launch-template.template.yaml

```diff
@@ -103,6 +103,14 @@ Resources:
                 - unzip
 
                 runcmd:
+                # add 4GB of swap space
```

Comment:
Prefer to move these lines to `ecs-additions-common.sh` and keep the LaunchTemplate UserData as minimal as possible. When the GWFCore stack is launched the `ecs-additions-*` scripts are "installed" in a CodeCommit repo. Updates to the repo trigger redeployment to an artifacts location that referenced by the LaunchTemplate. This enables users to customize provisioning steps without having to teardown and relaunch the GWFCore stack.

Reply:
Makes total sense. I will update it